tensorflow_serving/model_servers/load_model_prediction_service_impl.cc

```diff
@@ -0,0 +1,113 @@
+/* Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include <string>
+
+#include "tensorflow_serving/model_servers/load_model_prediction_service_impl.h"
+#include "tensorflow_serving/config/model_server_config.pb.h"
+
+#include "grpc/grpc.h"
+#include "tensorflow_serving/model_servers/grpc_status_util.h"
+#include "tensorflow_serving/servables/tensorflow/get_model_metadata_impl.h"
+#include "tensorflow_serving/model_servers/model_platform_types.h"
+
+namespace tensorflow {
+namespace serving {
+
+namespace {
+
+int DeadlineToTimeoutMillis(const gpr_timespec deadline) {
+  return gpr_time_to_millis(
+      gpr_time_sub(gpr_convert_clock_type(deadline, GPR_CLOCK_MONOTONIC),
+                   gpr_now(GPR_CLOCK_MONOTONIC)));
+}
+
+ModelServerConfig BuildSingleModelConfig(const string& model_name,
+                                         const string& model_base_path) {
+  ModelServerConfig config;
+  LOG(INFO) << "Building single TensorFlow model file config: "
+            << " model_name: " << model_name
+            << " model_base_path: " << model_base_path;
```

Comment:
Similarly, I think we'd like this to be on Debug level, or we don't log this at all - otherwise the logs will be filled with these entries.